src/osx/resources/universalJavaApplicationStub.sh

```diff
@@ -1,4 +1,5 @@
 #!/bin/sh
+export JAVA_HOME=$(/usr/libexec/java_home -v11)
```

Comment:
I proposed this as a quick solution. I don't think hardcoding the java version to use it's a good idea.

Reply:
it is not a good idea, bug it WORKS. THX